core/src/config.jl

```diff
@@ -249,6 +249,18 @@ const algorithms = Dict{String, Type}(
     "Euler" => Euler,
 )
 
+"""
+Check whether the given function has a method that accepts the given kwarg.
+Note that it is possible that methods exist that accept :a and :b individually,
+but not both.
+"""
+function function_accepts_kwarg(f, kwarg)::Bool
```

Comment:
Very meta, but nice improvement. Is `kwarg_decl` documented (i.e. public?)

Reply:
https://julialang.slack.com/archives/C7T968HRU/p1728982500911379?thread_ts=1728982500.911379&cid=C7T968HRU

Reply:
I don't think `Base.kwarg_decl` is documented, I can't find it and it doesn't have a docstring.

Reply:
Well that's bad, but then again, it seems this method has been there for at least 5 years, so let's use it. Nice find.